test/stdlib/MultilineString.swift

```diff
@@ -0,0 +1,124 @@
+// RUN: %target-run-stdlib-swift | %FileCheck %s
```

Comment:
What is the easiest way to test the tests for you? Is there a command to run an individual test?